pkgs/by-name/dw/dwlmsg/default.nix

```diff
@@ -0,0 +1,16 @@
+let
```

Comment:
i don't think this should exist here? at least i don't see any other packages with it you should be able to build it without this by running `nix-build -A dwlmsg`

Reply:
the default.nix here defines what dwlmsg means to nix-build.
I'm not sure how I would do this without the default.nix.

the package.nix when included in your configuration.nix builds and installs without the default.nix at all

Reply:
the package.nix file i used was directly borrowed from DWL & that didn't have a default.nix file, I wouldn't be upset if we decided that this didn't get merged.